Remotion/Data/DomainObjects/Infrastructure/DomainObjectTransactionContextStruct.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Remotion.Data.DomainObjects.DataManagement;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Infrastructure
+{
+  public readonly struct DomainObjectTransactionContextStruct
```

Comment:
This struct and all its members need XML docs. We can take the docs from `IDomainObjectTransactionContext` since we probably delete this anyway.